docker/Dockerfile.gpu

```diff
@@ -36,6 +36,13 @@ COPY docker/poetry-env.sh docker/poetry-env.sh
 # Build poetry virtual environment
 RUN bash docker/poetry-env.sh PYTHON_BINARY=$PYTHON_BINARY GEN_PYTORCH_EXT="$GEN_PYTORCH_EXT" GEN_OPENGL_EXT="$GEN_OPENGL_EXT"
 
+# Build FFmpeg as shared lib
+RUN git clone https://github.com/FFmpeg/FFmpeg.git && \
```

Comment:
Could you remind me again. What was the use for these libs ?

Reply:
I think @rarzumanyan can answer this question better. I would say ffmpeg delivers all the avcodec libs and headers that are needed.

Reply:
We do install them [here](https://github.com/NVIDIA/VideoProcessingFramework/blob/master/docker/Dockerfile.gpu#L18) both in build and dev stage

Reply:
Yes but the current master does not build with those apt packages. This is why i build from source.  https://github.com/NVIDIA/VideoProcessingFramework/issues/355#issuecomment-1187572408